ui-particles-angular/projects/ui-policy-studio-angular/src/lib/gio-policy-studio.spec.ts

```diff
@@ -416,6 +451,49 @@ describe('GioPolicyStudioModule', () => {
     expect(await detailsHarness.matchText(/webhook/)).toEqual(true);
   });
 
+  it('should edit flow into plan for proxy API', async () => {
+    const planFooFlows = [fakeChannelFlow({ name: 'Foo flow 1' }), fakeChannelFlow({ name: 'Foo flow 2' })];
```

Comment:
hum you need to use `fakeHttpFlow` to make consistant test. Channel is only for MESSAGE api type